### PR TITLE
updated the version and made the cluster name optional

### PR DESCRIPTION
**Pull request by bmason42**
updated the version and made the cluster name optional

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.0
\ No newline at end of file
+2.0
```
Comment (coffeefreak101) [inline version.txt:1]: Do we need a major version change? I would think that making a required field optional would only need a minor version change. I don't really care too much though.

**Review by coffeefreak101 — APPROVED** [review]

File: astraconnector_operator.yaml [hunk 2]
```diff
@@ -462,7 +461,7 @@ spec:
         - --leader-elect
         command:
         - /manager
-        image: netapp/astra-connector-operator:1.0.202209131709
+        image: netapp/astra-connector-operator:0.0.1
```
Comment (raghu-sarangapani) [inline astraconnector_operator.yaml:464]: is this intentional?

File: version.txt (hunk 1 above)
Comment (bmason42) [reply version.txt:1]: Do we ?  Probably not.  I am planning on a few more changes and just thought to bump it to 2

File: astraconnector_operator.yaml (hunk 2 above)
Comment (bmason42) [reply astraconnector_operator.yaml:464]: no, not sure how that happened.  In fact, it should go to 2.0

Comment (bmason42) [reply astraconnector_operator.yaml:464]: the regenerate script sets that back to 0.0.1

Comment (bmason42) [reply astraconnector_operator.yaml:464]: fixed now

**Pull request merged**